bedboss/bedclassifier/bedclassifier.py

```diff
@@ -0,0 +1,234 @@
+
+        num_cols = len(df.columns)
+        bedtype = 0
+
+        for col in df:
+            if col <= 2:
+                if col == 0:
+                    if df[col].dtype == "O":
+                        bedtype += 1
+                    elif df[col].dtype == "int" or df[col].dtype == "float":
+                        bedtype += 1
+                    else:
+                        if no_fail:
+                            _LOGGER.warning(
+                                f"Bed type could not be determined at column {0} with data type: {df[col].dtype}"
+                            )
+                            return "unknown_bedtype"
+                        else:
+                            raise BedTypeException(
+                                reason=f"Bed type could not be determined at column {0} with data type: {df[col].dtype}"
+                            )
+
+                else:
+                    if df[col].dtype == "int" and (df[col] >= 0).all():
+                        bedtype += 1
+                    else:
+                        if no_fail:
+                            _LOGGER.warning(
+                                f"Bed type could not be determined at column {col} with data type: {df[col].dtype}"
+                            )
+                            return "unknown_bedtype"
+                        else:
+                            raise BedTypeException(
+                                reason=f"Bed type could not be determined at column 0 with data type: {df[col].dtype}"
+                            )
+            else:
+                if col == 3:
+                    if df[col].dtype == "O":
+                        bedtype += 1
+                    else:
+                        n = num_cols - bedtype
+                        return f"bed{bedtype}+{n}"
+                elif col == 4:
+                    if df[col].dtype == "int" and df[col].between(0, 1000).all():
+                        bedtype += 1
+                    else:
+                        n = num_cols - bedtype
+                        return f"bed{bedtype}+{n}"
+                elif col == 5:
+                    if df[col].isin(["+", "-", "."]).all():
+                        bedtype += 1
+                    else:
+                        n = num_cols - bedtype
+                        return f"bed{bedtype}+{n}"
+                elif 6 <= col <= 8:
+                    if df[col].dtype == "int" and (df[col] >= 0).all():
+                        bedtype += 1
+                    else:
+                        n = num_cols - bedtype
+                        return f"bed{bedtype}+{n}"
+                elif col == 9:
+                    if df[col].dtype == "int":
+                        bedtype += 1
+                    else:
+                        n = num_cols - bedtype
+                        if "broadpeak" in bed or "broadPeak" in bed:
+                            return f"broadPeak,bed{bedtype}+{n}"
```

Comment:
This is incorrect
We have to return or tuple, or `bed{bedtype}+{n}`, because it is 2 different values and information.
Additionally, this method should always have same return type

Reply:
I went back and forth on this. I didn't want to just return broadPeak without the context of how many columns it has.
But I didn't want to return a tuple because the majority of the bedfiles will (this is an assumption) be classified in the format `bed{bedtype}+{n}`.

Reply:
I understand it, but it's not standard, and we will have to parse this string later in processing, it means additional work to be done. In this case, I would just completely delete broad and narrowpeak

Reply:
This should be 2 separate columns in our database